### PR TITLE
Replace Goerli network with Sepolia in the test TestBlockTransactionCount

### PR DESCRIPTION
**Pull request by stefanalfbo**
This PR is fixing the issue #1792, includes making a copy of necessary testdata used from the goerli directory to the sepolia directory.

### CODE REVIEW

File: clients/feeder/testdata/sepolia/block/485004.json [hunk 1]
```diff
@@ -0,0 +1,2040 @@
+{
+    "block_hash": "0x7f9413a02d787cb52046925414b4f4f4cfde93b4a96f92fa7c57eb0ac302f3",
```
Comment (rianhughes) [inline clients/feeder/testdata/sepolia/block/485004.json:2]: Where did you get this test data? As far as I can see Sepolia block height is ~56,000 not 485,004

Comment (rianhughes) [reply clients/feeder/testdata/sepolia/block/485004.json:2]: Oh right, you just copied the Goerli block data. I think you should add new test data, eg get a Sepolia block + signature

Comment (rianhughes) [reply clients/feeder/testdata/sepolia/block/485004.json:2]: https://alpha-sepolia.starknet.io/feeder_gateway/get_block?blockNumber=56377

Comment (rianhughes) [reply clients/feeder/testdata/sepolia/block/485004.json:2]: https://alpha-sepolia.starknet.io/feeder_gateway/get_signature?blockNumber=56377

Comment (rianhughes) [reply clients/feeder/testdata/sepolia/block/485004.json:2]: You should do the same for your other issue https://github.com/NethermindEth/juno/pull/1793

**Comment by stefanalfbo** [comment]
Ok, I will do that for both PR\s, thanks for the links and the feedback!

**Review by rianhughes — APPROVED** [review]

**Pull request merged**